visualization/dashboard/src/MLIDashboard/ExplanationDashboard.tsx

```diff
@@ -70,6 +70,13 @@ export class ExplanationDashboard extends React.Component<IExplanationDashboardP
         "ICE"
     ];
 
+    private static initializeIcons(props: IExplanationDashboardProps): void {
+        if (ExplanationDashboard.iconsInitialized === false && props.shouldInitializeIcons !== false) {
```

Comment:
it really should be:
(!ExplanationDashboard.iconsInitialized && props.shouldInitializeIcons)
but I guess that's not how typescript works?

Reply:
lol, yes this is specifically for the typescript/js case that undefined evaluates to false. In the case that it's not set, we should defualt to initializing since this is the least damaging choice